releasenotes/notes/apm-update-peer-tags-c4ca55ac8e190114.yaml

```diff
@@ -0,0 +1,3 @@
+enhancements:
```

Comment:
Not sure release notes are needed for this PR? If not just add the label `no-changelog`

Reply:
Makes sense. Made a small suggestion but otherwise LGTM.